azkaban-common/src/main/java/azkaban/utils/OsMemoryUtil.java

```diff
@@ -0,0 +1,110 @@
+package azkaban.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Utility class for getting system memory information
+ */
+class OsMemoryUtil {
```

Comment:
I found the suggested name harder to read.
I am fine with the change if it is part of the naming convention. Could you point me to the naming convention doc that mandates this?